src/login/logind-dbus.c

```diff
@@ -3789,6 +3789,12 @@ int match_job_removed(sd_bus_message *message, void *userdata, sd_bus_error *err
                 return 0;
         }
 
+        if (sd_bus_message_is_signal(message, NULL, "JobRemovedEx")) {
```

Comment:
we expect logind and machined to be update along with pid 1, hence just rework to use JobRemovedEx unconditionally